test/run-spec-wasm2c.py

```diff
@@ -386,6 +386,7 @@ def main(args):
             error_cmdline=options.error_cmdline)
         wast2json.verbose = options.print_cmd
         wast2json.AppendOptionalArgs({'-v': options.verbose})
+        wast2json.AppendArg('--enable-multi-memory')
```

Comment:
Again, this should probably go in the test file

Reply:
Can do (if I understood your comment above correctly) -- we'll also have to plumb through the selectable features in run-spec-wasm2c.py. It's not as sophisticated as the interpreter runner and I don't think it supports command-line enable/disable arguments in the test file yet.

Reply:
I think it probably makes sense to pass features through to wasm2c, yes.   Mostly because all the other wabt tools do that.